cmd/experimental/kjobctl/apis/v1alpha1/application_profile_types.go

```diff
@@ -69,6 +69,8 @@ type SupportedMode struct {
 	// requiredFlags point which cli flags are required to be passed in order to fill the gaps in the templates.
 	// Possible values are cmd, parallelism, completions, request, localqueue.
 	//
+	// cmd and requests values are going to be added only to the first primary container.
```

Comment:
The flags should be specific to the used profile. For example, parallelism doesn't make sense for interactive. While it doesn't have to be checked at the CRD level, we probably want to check when running the app.

Reply:
We are checking that the required flags are provided , here: https://github.com/kubernetes-sigs/kueue/blob/398545f971a9d5ecb83d69f410f606525aaa8814/cmd/experimental/kjobctl/pkg/builder/builder.go#L169